tests/test_api.py

```diff
@@ -777,3 +783,314 @@ def test_gender_acc_invalid_range(self):
 
         with self.assertRaisesRegex(InvalidValueError, msg):
             api.update_profile(uuid, gender='male', gender_acc=101)
+
+
+class TestEnroll(TestCase):
+    """Unit tests for enroll"""
+
+    def test_enroll(self):
+        """Check whether it adds an enrollments to a unique identity and a organization"""
```

Comment:
typo -> an organization

Reply:
It should be fixed with the latest changes.